components/Container/Container.js

```diff
@@ -42,14 +42,21 @@ function Container({
 
   return (
     <div
-      className={classNames(className, styles.Container, styles[theme], {
-        [styles.fullViewportHeight]: isFullViewportHeight,
-      })}
+      className={classNames(
+        className,
+        'bg-center bg-no-repeat bg-cover flex items-center justify-center min-h-[250px] w-full fill-current',
+        styles[theme],
```

Comment:
```suggestion
        'bg-secondary color-white [& svg]:fill-white': theme === 'secondary',
```

Here's how you'd convert `styles[theme]` for when `theme === 'secondary'`. You'll still need to add code for when `theme === 'white'` and when `theme === 'gray'`